crates/typst/src/math/fragment.rs

```diff
@@ -231,6 +247,7 @@ pub struct GlyphFragment {
     pub dests: SmallVec<[Destination; 1]>,
     pub hidden: bool,
     pub limits: Limits,
+    pub stretch: Option<StretchInfo>,
```

Comment:
the proliferation of fields on this struct is really unfortunate

Reply:
Indeed, but necessary..